LABTASKS.md

```diff
@@ -107,59 +111,47 @@ Please read carefully the list of technical requirements below and follow it pre
-      > designs for each type of device. To see how your page would look on some phones
-      > or tablets:
-      >
-      > Open your page in Chrome > right-click the page > click "inspect element" > click
-      > the little phone symbol in the top-left corner. This will show roughly what your
-      > site would look like on the chosen device. In Firefox, click on the Developer
-      > button and choose Responsive Design mode. Or just resize your browser
-      > window so that it's really narrow and see how it behaves.
-  - Your CSS can leverage the fact that the page has the following
-    **elements of interest**; see [the links in the README](./README.md) for support
-    in understanding how to use CSS to alter the look of the page. You should
-    particularly care about these features that were included to make the CSS more fun:
-    - A **navigation bar.** Actually, this is just a list that is presented as a
-      navigation bar in `style1.css' It must behave reasonably when you resize the window.
-      In your style files, it may be horizontal or vertical, but should differ
-      in some meaningful way between your two CSS files.
-    - Four images of the faculty members **each of which has a div with a unique id**
-    - Course requirements with a variety of parts that have class attribute values
-    - Four faculty members in a list with their names and office locations in div classes
-    - Multiple divs that have unique ids **but have the same class attribute value**
-    - Multiple lists, both ordered and unordered **so that you may handle lists
-      in different ways** and show some cool css features
+- Your CSS must make the page **look reasonable and be relatively easy to navigate.**
+  We will take points off for features that lead to a problematic user experience. Take
+  care that your design avoids common pitfalls.
+  - Avoid links that are hard to see.
+  - Avoid unpleasant color combinations.
+  - Avoid fonts that are hard to read.
+    > Pro tip: Fonts specified in CSS can be different in different browsers! Just
+    > because `font-family: "Comic Sans"` looks sweet in Internet Explorer 5 (which you
+    > better not be using) does not mean it will in Chrome or Firefox.
+    >
+    > You can use nifty fonts from places like
+    > [Google Fonts](https://fonts.google.com); this will require
+    > an (acceptable) change to the HTML file to load in the necessary
+    > font files. Tools like [FontJoy](http://fontjoy.com) can be useful for
+    > helping find font combinations that look good together.
+  - Your page must look reasonable in all standard browsers (recent Chrome, Firefox,
+    Safari, IE, Edge) and adapt the layout so that elements remain accessible and visible when the window is resized.
+    > To see how your page would look on some phones or tablets:
+    >
+    > Open your page in Chrome > right-click the page > click "Inspect" > click
```

Comment:
That'd certainly work too. I'd prefer to only give one set of instructions, though, just to avoid over complicating things. 🙂 No need to give people more than one way to do it!

Reply:
`Command+shift+i` didn't work for me on Chrome on my Mac just now, although F12. This is the problem with these kinds of shortcuts; other software can block or interfere with them. I figure the kind of people who will be likely to remember and use a shortcut are also the kind of people who can figure out what the shortcuts are. 😜